website/common/locales/en@pirate/messages.json

```diff
@@ -30,6 +30,7 @@
     "messageAlreadyPurchasedGear": "Ye purchased this gear in th' past, but do not currently own it. Ye can buy it again in th' rewards column on th' tasks page.",
     "messageAlreadyOwnGear": "Ye already own this item. Equip it by going to th' equipment page.",
     "messageHealthAlreadyMax": "You already have maximum health.",
+    "messageHealthAlreadyMin": "You already have the minimum health.",
```

Comment:
Can you please remove the change from this file and also the two `es` files? Translations are handled through Transifex and it's best to not overwrite them in GitHub. See the _README_FIRST.md file in this directory for more details.